content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,337 @@
+=============================
+Lost leads reactivation email
+=============================
+
+In Odoo, lost leads are removed from the active *CRM* pipeline, but can still be targeted with the
+*Email Marketing* application for strategic campaign use such as lost leads reactivation.
+
+A lost leads reactivation email looks at the leads that were lost during a specific period of time,
+and uses a few custom filters and lost reasons to exclude undesirable leads from the mailing list.
+
+Once a lost leads reactivation email is complete, it can be sent as is, modified and sent to
+different groups for A/B testing, or saved as a template for later.
+
+.. example::
+   A warehouse has leftover merchandise from a limited run of items from last year. To help clear
+   out the excess inventory, the warehouse manager creates a lost leads email to reach out to old
+   opportunities that were lost, and inform them that the limited merchandise is back in stock.
+
+   The warehouse manager uses the following filters for a lost leads email:
+
+   - :guilabel:`Blacklist` *is* `not set`
+   - :guilabel:`Created on` *>=* `01/01/2024 00:00:01`
+   - :guilabel:`Stage` *is not in* `New`, `Qualified`, or `Won`
+   - :guilabel:`Lost Reason` *is in* `Not enough stock`
+   - and either :guilabel:`Active` *is* `set` or `not set`
+
+   .. image:: lost_leads_email/example.png
+      :align: center
+      :alt: A lost leads reactivation filter list excluding lost reasons like Too Expensive.
+
+.. tip::
+   As filters are added and removed, pay attention to the :guilabel:`# record(s)` value below the
+   filtering section. This value indicates the total number of records that match the current
+   criteria.
+
+   To view a list of all matching records, click the :guilabel:`# record(s)` text.
+
+   .. image:: lost_leads_email/records.png
+      :align: center
+      :alt: The # record(s) text is located below the list of Recipient filters.
+
+.. _create_email:
+
+Minimum requirements
+====================
+
+In order to create and deliver a lost leads reactivation email campaign, the *CRM* and *Email
+Marketing* applications must be :doc:`installed <../../general/apps_modules>` and configured.
+
+Here are the minimum necessary filters that pertain to a lost leads reactivation mailing campaign:
+
+- A **Blacklist** filter to exclude unsubscribed recipients
+- A **Created on** filter to target leads that were lost during a specific period of time
+- **Stage** filter(s) to exclude leads that were already won or are still active in new stages of
+  the sales pipeline (i.e., *New*, *Qualified*, etc.). These values will be different per
+  organization, however it's minimally viable to exclude all the leads in the *Won* stage.
+- One or more **Lost Reason** filters to exclude undesired leads, such as duplicate, spam, or
+  irrelevant records.
+- A pair of **Active** filters to target *both* active and inactive leads
+
+Add the necessary filters
+=========================
```

Comment:
I see what you mean about being more succinct, especially since it's a heading.
I do, however, disagree on this one since these are *specific* filters, and are not optional to send this campaign; I want to indicate this and believe removing the article might imply some optionality here, when that's actually not the case.